flags/14.0/vtgate.txt

```diff
@@ -1,48 +1,47 @@
 Usage of vtgate:
-      --allowed_tablet_types value                                       Specifies the tablet types this vtgate is allowed to route queries to
+      --allowed_tablet_types []topodatapb.TabletType                     Specifies the tablet types this vtgate is allowed to route queries to.
       --alsologtostderr                                                  log to standard error as well as files
       --buffer_drain_concurrency int                                     Maximum number of requests retried simultaneously. More concurrency will increase the load on the PRIMARY vttablet when draining the buffer. (default 1)
-      --buffer_implementation string                                     Allowed values: healthcheck (legacy implementation), keyspace_events (default) (default keyspace_events)
+      --buffer_implementation string                                     Allowed values: healthcheck (legacy implementation), keyspace_events (default) (default "keyspace_events")
       --buffer_keyspace_shards string                                    If not empty, limit buffering to these entries (comma separated). Entry format: keyspace or keyspace/shard. Requires --enable_buffer=true.
       --buffer_max_failover_duration duration                            Stop buffering completely if a failover takes longer than this duration. (default 20s)
       --buffer_min_time_between_failovers duration                       Minimum time between the end of a failover and the start of the next one (tracked per shard). Faster consecutive failovers will not trigger buffering. (default 1m0s)
       --buffer_size int                                                  Maximum number of buffered requests in flight (across all ongoing failovers). (default 1000)
       --buffer_window duration                                           Duration for how long a request should be buffered at most. (default 10s)
       --catch-sigpipe                                                    catch and ignore SIGPIPE on stdout and stderr if specified
-      --cell string                                                      cell to use (default test_nj)
```

Comment:
@harshit-gangal, what is "test_nj", the default for `-cell`?

Reply:
This is a default defined as such for historical reasons. Plan is to mark this flag as required and remove this default.

Reply:
sg